backend/src/api/app.ts

```diff
@@ -308,6 +309,19 @@ authenticatedRoute.get('/users', handlerToExpress(users.list));
 authenticatedRoute.post('/users', handlerToExpress(users.invite));
 authenticatedRoute.delete('/users/:userId', handlerToExpress(users.del));
 
+authenticatedRoute.post(
+  '/pe-reports/export',
+  handlerToExpress(peReports.export_pe_report)
+);
+authenticatedRoute.post(
+  '/pe-reports/upload',
```

Comment:
```suggestion
  '/pe-reports',
```

Reply:
removed functionality to upload from frontend